src/main/java/com/gluonhq/substrate/model/Triplet.java

```diff
@@ -35,6 +35,8 @@
 
 public class Triplet {
 
+    private static final String OS_NAME  = System.getProperty("os.name").toLowerCase(Locale.ROOT);
```

Comment:
Can you add a comment here that makes it clear this is evaluated at build-time?

Reply:
Added